### PR TITLE
Make sure default timeout is in ms, and rename DEFAULT_ADB_TIMEOUT

### PR DESCRIPTION
**Pull request by gdaniels**
https://github.com/square/spoon/pull/314#discussion_r51683768

### CODE REVIEW

File: spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java [hunk 1]
```diff
@@ -288,7 +288,7 @@ private SpoonDeviceRunner getTestRunner(String serial, SpoonInstrumentationInfo
     private String methodName;
     private boolean noAnimations;
     private IRemoteAndroidTestRunner.TestSize testSize;
-    private int adbTimeout = DEFAULT_ADB_TIMEOUT;
+    private int adbTimeout = DEFAULT_ADB_TIMEOUT_SEC * 1000;
```
Comment (JakeWharton) [inline spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java]: Rename to append `Millis`?

**Comment by JakeWharton** [comment]
LGTM otherwise

File: spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java (hunk 1 above)
Comment (gdaniels) [inline spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java]: Only issue with that is then either the public API needs to change (setAdbTimeoutMillis()) or the setter API won't match the field name.  I suppose we could add setAdbTimeoutMillis() and deprecate setAdbTimeout()...  or just be ok with the method/field naming difference.  Thoughts?

Comment (JakeWharton) [inline spoon-runner/src/main/java/com/squareup/spoon/SpoonRunner.java]: I'm fine with the method name staying the same. It has a whole slew of problems I'm not interested in fixing at this point.

**Comment by gdaniels** [comment]
All set with the rename!

**Comment by JakeWharton** [comment]
Thanks!

**Pull request merged**